new/INS1003MEAG.xml

```diff
@@ -0,0 +1,57 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="INS1003MEAG" xml:lang="en" type="ins">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Private Collection of Məntasnot  ʾAbbuhay (Gondar Bātā Church) </title>
```

Comment:
1) Is LOC1676Baata used below the same as INS0995GondarBLM for Gondar Baʾātā Lamāryām? (Do we have then a double record for the same place?) 2) "Private collection of Məntasnot  ʾAbbuhay (Gondar Bātā Church)" - It is not quite clear what (Gondar Bātā Church) means here. I think you can say simply "Private collection of Məntasnot  ʾAbbuhay", and show his relation to the church trough a commentary and relations.

Reply:
1) Denis, LOC and INS are not doublets, this was initially intended. All places are LOC, additionally manuscript-holding repositories have also an INS-type ID. If the place is the same this should be recorded in a relation.

2) Agree.

Reply:
I do not know the source for the coordinates, they must be wrong, the description refers to BM at Gondar and has the bibliographic reference to Gondar, the coordinates point to Ambobar (Maryam), must be a confusion, LOC1676Baata should be corrected

Reply:
No sorry my wrong the coordinates are quite close

Reply:
Close (occassionally pointing to a mosque), but it would be better if they could lead closer to Ba'ata. Also, since Addisie is already at that, she could insert coordinates to INS0995GondarBLM, couldn't she?

Reply:
![image](https://github.com/BetaMasaheft/Institutions/assets/17987728/bd6032d3-3af7-478f-8b34-c94727edcb4e)


the decimal coordinates can be corrected slightly to 12.6020 37.46100 to match

Reply:
The LOC record has an invisible comment `<!--                  geo coord refer to Gondar as a whole-->` so can be disregarded, have created PRs